src/harness/fourslash.ts

```diff
@@ -20,13 +20,16 @@
 /// <reference path="fourslashRunner.ts" />
 
 namespace FourSlash {
+    const _fs = require("fs");

```

Comment:
You can't use 'fs' in here (or any other Node.js module), as fourslash tests need to run in other environments (e.g. the browser). You'll notice there are no other 'require' calls in this code.